client/src/components/Search/styled.js

```diff
@@ -0,0 +1,37 @@
+import styled from 'styled-components';
```

Comment:
for the sake of consistency across the app, let's use `makeStyles` to handle the styling

Reply:
Got it, I will convert them to makeStyles.